gh-pages-stub/_data/service_list.yml

```diff
@@ -1,25 +1,30 @@
 docs:
-  - group: Communication
-    groupitems:
-      - doc: KPC
-      - doc: CM
-      - doc: EPL
-  - group: Data collection
-    groupitems:
-      - doc: DCX
-      - doc: EPTS
-      - doc: WSTS
-  - group: Identity management
-    groupitems:
-      - doc: EPR
-      - doc: EPMX
-  - group: Configuration management
-    groupitems:
-      - doc: CMX
-      - doc: ECR
-  - group: Front end
-    groupitems:
-      - doc: WD
-  - group: Service infrastructure
-    groupitems:
-      - doc: SCR
+- group: Communication
+  groupitems:
+  - doc: KPC
+  - doc: CM
+  - doc: EPL
+- group: Data collection
+  groupitems:
+  - doc: DCX
+  - doc: EPTS
+  - doc: WSTS
+- group: Identity management
+  groupitems:
+  - doc: EPR
+  - doc: EPMX
+- group: Configuration management
+  groupitems:
+  - doc: CMX
+  - doc: ECR
+- group: Over-the-air updates
```

Comment:
Let's have a more general name for this group, e. g. Software updates, as we might have other types of updates besides OTA in future.

Reply:
Updated.